adding chaos-engineering to topics (#2636)
github/explore: topics/chaos-engineering/index.md

---

> aliases: chaos-engineering, site-reliability-engineering

Reviewer: this alias list cannot include the topic name
```suggestion
aliases: site-reliability-engineering
```

Author: sure, i will make this change @kenyonj

display_name: Chaos Engineering

> logo:

Reviewer: ```suggestion
```

---
aliases: site-reliability-engineering
display_name: Chaos Engineering
related: sre, testing

> short_description: Chaos engineering is the process of testing a distributed computing system to ensure that it can withstand unexpected disruptions

Reviewer: Perhaps this will work?
```suggestion
short_description: The process of testing distributed computing systems to ensure that they can withstand unexpected disruptions.
```

> short_description: Chaos engineering is the process of testing a distributed computing system to ensure that it can withstand unexpected disruptions.

Reviewer: ```suggestion
short_description: The process of testing distributed computing systems to ensure that they can withstand unexpected disruptions.
```
